From 736c1704505044b7dce5682dea3efb79bfcca40e Mon Sep 17 00:00:00 2001
From: Tri Luong <tri.luong.te@rvc.renesas.com>
Date: Tue, 7 Aug 2018 01:58:32 +0700
Subject: [PATCH 604/628] pfc-r8a7744: Remove wrong definition

SEL_TSIF0, TS_SDATA0, TS_SCK0, TS_SDEN0, TS_SPSYNC0 pin fucntion.

Signed-off-by: Tri Luong <tri.luong.te@rvc.renesas.com>
---
 drivers/pinctrl/sh-pfc/pfc-r8a7744.c | 117 +++++++++++++++--------------------
 1 file changed, 50 insertions(+), 67 deletions(-)

diff --git a/drivers/pinctrl/sh-pfc/pfc-r8a7744.c b/drivers/pinctrl/sh-pfc/pfc-r8a7744.c
index a80b7a0..86b5ab0 100644
--- a/drivers/pinctrl/sh-pfc/pfc-r8a7744.c
+++ b/drivers/pinctrl/sh-pfc/pfc-r8a7744.c
@@ -220,15 +220,15 @@ enum {
 	FN_SSI_SCK4,
 	FN_SSI_WS4,
 	FN_SSI_SDATA4, FN_MSIOF2_SCK_D,
-	FN_SSI_SCK5, FN_MSIOF1_SCK_C, FN_TS_SDATA0,
+	FN_SSI_SCK5, FN_MSIOF1_SCK_C,
 	FN_MSIOF2_SYNC_D, FN_VI1_R2_B,
 
 	/* IPSR5 */
-	FN_SSI_WS5, FN_MSIOF1_SYNC_C, FN_TS_SCK0,
+	FN_SSI_WS5, FN_MSIOF1_SYNC_C,
 	FN_MSIOF2_TXD_D, FN_VI1_R3_B,
-	FN_SSI_SDATA5, FN_MSIOF1_TXD_C, FN_TS_SDEN0,
+	FN_SSI_SDATA5, FN_MSIOF1_TXD_C,
 	FN_MSIOF2_SS1_D, FN_VI1_R4_B,
-	FN_SSI_SCK6, FN_MSIOF1_RXD_C, FN_TS_SPSYNC0,
+	FN_SSI_SCK6, FN_MSIOF1_RXD_C,
 	FN_MSIOF2_RXD_D, FN_VI1_R5_B,
 	FN_SSI_WS6, FN_MSIOF2_SS2_D, FN_VI1_R6_B,
 	FN_SSI_SDATA6, FN_VI1_R7_B,
@@ -308,10 +308,10 @@ enum {
 	FN_CAN0_RX, FN_RX3_B, FN_SDA2_B,
 	FN_DU1_DISP,
 	FN_DU1_CDE, FN_PWM4_B,
-	FN_VI0_CLKENB, FN_TX4, FN_SCIFA4_TXD, FN_TS_SDATA0_D,
-	FN_VI0_FIELD, FN_RX4, FN_SCIFA4_RXD, FN_TS_SCK0_D,
-	FN_VI0_HSYNC_N, FN_TX5, FN_SCIFA5_TXD, FN_TS_SDEN0_D,
-	FN_VI0_VSYNC_N, FN_RX5, FN_SCIFA5_RXD, FN_TS_SPSYNC0_D,
+	FN_VI0_CLKENB, FN_TX4, FN_SCIFA4_TXD,
+	FN_VI0_FIELD, FN_RX4, FN_SCIFA4_RXD,
+	FN_VI0_HSYNC_N, FN_TX5, FN_SCIFA5_TXD,
+	FN_VI0_VSYNC_N, FN_RX5, FN_SCIFA5_RXD,
 	FN_VI0_DATA3_VI0_B3, FN_SCIF3_SCK_B, FN_SCIFA3_SCK_B,
 	FN_VI0_G0, FN_SCL8, FN_SCL4,
 	FN_HCTS2_N, FN_SCIFB2_CTS_N, FN_ATAWR1_N,
@@ -330,11 +330,11 @@ enum {
 	FN_VI0_G6, FN_VI2_CLK,
 	FN_VI0_G7, FN_VI2_DATA0,
 	FN_VI0_R0, FN_VI2_DATA1,
-	FN_TS_SDATA0_C, FN_ATACS11_N,
+	FN_ATACS11_N,
 	FN_VI0_R1, FN_VI2_DATA2,
-	FN_TS_SCK0_C, FN_ATAG1_N,
-	FN_VI0_R2, FN_VI2_DATA3, FN_TS_SDEN0_C,
-	FN_VI0_R3, FN_VI2_DATA4, FN_TS_SPSYNC0_C,
+	FN_ATAG1_N,
+	FN_VI0_R2, FN_VI2_DATA3,
+	FN_VI0_R3, FN_VI2_DATA4,
 	FN_VI0_R4, FN_VI2_DATA5, FN_TX0_C, FN_SCL1_D,
 
 	/* IPSR11 */
@@ -342,10 +342,10 @@ enum {
 	FN_VI0_R6, FN_VI2_DATA7, FN_TX1_C, FN_SCL4_B,
 	FN_VI0_R7, FN_RX1_C, FN_CAN0_RX_E,
 	FN_SDA4_B, FN_HRX1_D, FN_SCIFB0_RXD_D,
-	FN_VI1_HSYNC_N, FN_AVB_RXD0, FN_TS_SDATA0_B, FN_TX4_B, FN_SCIFA4_TXD_B,
-	FN_VI1_VSYNC_N, FN_AVB_RXD1, FN_TS_SCK0_B, FN_RX4_B, FN_SCIFA4_RXD_B,
-	FN_VI1_CLKENB, FN_AVB_RXD2, FN_TS_SDEN0_B,
-	FN_VI1_FIELD, FN_AVB_RXD3, FN_TS_SPSYNC0_B,
+	FN_VI1_HSYNC_N, FN_AVB_RXD0, FN_TX4_B, FN_SCIFA4_TXD_B,
+	FN_VI1_VSYNC_N, FN_AVB_RXD1, FN_RX4_B, FN_SCIFA4_RXD_B,
+	FN_VI1_CLKENB, FN_AVB_RXD2,
+	FN_VI1_FIELD, FN_AVB_RXD3,
 	FN_VI1_CLK, FN_AVB_RXD4, FN_VI1_DATA0, FN_AVB_RXD5,
 	FN_VI1_DATA1, FN_AVB_RXD6, FN_VI1_DATA2, FN_AVB_RXD7,
 	FN_VI1_DATA3, FN_AVB_RX_ER, FN_VI1_DATA4, FN_AVB_MDIO,
@@ -445,7 +445,6 @@ enum {
 	FN_SEL_VI1_0, FN_SEL_VI1_1, FN_SEL_VI1_2,
 	FN_SEL_TMU1_0, FN_SEL_TMU1_1,
 	FN_SEL_LBS_0, FN_SEL_LBS_1, FN_SEL_LBS_2, FN_SEL_LBS_3,
-	FN_SEL_TSIF0_0, FN_SEL_TSIF0_1, FN_SEL_TSIF0_2, FN_SEL_TSIF0_3,
 	FN_SEL_SOF0_0, FN_SEL_SOF0_1, FN_SEL_SOF0_2,
 
 	/* MOD_SEL2 */
@@ -578,15 +577,15 @@ enum {
 	SSI_SCK4_MARK,
 	SSI_WS4_MARK,
 	SSI_SDATA4_MARK, MSIOF2_SCK_D_MARK,
-	SSI_SCK5_MARK, MSIOF1_SCK_C_MARK, TS_SDATA0_MARK,
+	SSI_SCK5_MARK, MSIOF1_SCK_C_MARK,
 	MSIOF2_SYNC_D_MARK, VI1_R2_B_MARK,
 
 	/* IPSR5 */
-	SSI_WS5_MARK, MSIOF1_SYNC_C_MARK, TS_SCK0_MARK,
+	SSI_WS5_MARK, MSIOF1_SYNC_C_MARK,
 	MSIOF2_TXD_D_MARK, VI1_R3_B_MARK,
-	SSI_SDATA5_MARK, MSIOF1_TXD_C_MARK, TS_SDEN0_MARK,
+	SSI_SDATA5_MARK, MSIOF1_TXD_C_MARK,
 	MSIOF2_SS1_D_MARK, VI1_R4_B_MARK,
-	SSI_SCK6_MARK, MSIOF1_RXD_C_MARK, TS_SPSYNC0_MARK,
+	SSI_SCK6_MARK, MSIOF1_RXD_C_MARK,
 	MSIOF2_RXD_D_MARK, VI1_R5_B_MARK,
 	SSI_WS6_MARK, MSIOF2_SS2_D_MARK, VI1_R6_B_MARK,
 	SSI_SDATA6_MARK, VI1_R7_B_MARK,
@@ -668,10 +667,10 @@ enum {
 	CAN0_RX_MARK, RX3_B_MARK, SDA2_B_MARK,
 	DU1_DISP_MARK,
 	DU1_CDE_MARK, PWM4_B_MARK,
-	VI0_CLKENB_MARK, TX4_MARK, SCIFA4_TXD_MARK, TS_SDATA0_D_MARK,
-	VI0_FIELD_MARK, RX4_MARK, SCIFA4_RXD_MARK, TS_SCK0_D_MARK,
-	VI0_HSYNC_N_MARK, TX5_MARK, SCIFA5_TXD_MARK, TS_SDEN0_D_MARK,
-	VI0_VSYNC_N_MARK, RX5_MARK, SCIFA5_RXD_MARK, TS_SPSYNC0_D_MARK,
+	VI0_CLKENB_MARK, TX4_MARK, SCIFA4_TXD_MARK,
+	VI0_FIELD_MARK, RX4_MARK, SCIFA4_RXD_MARK,
+	VI0_HSYNC_N_MARK, TX5_MARK, SCIFA5_TXD_MARK,
+	VI0_VSYNC_N_MARK, RX5_MARK, SCIFA5_RXD_MARK,
 	VI0_DATA3_VI0_B3_MARK, SCIF3_SCK_B_MARK, SCIFA3_SCK_B_MARK,
 	VI0_G0_MARK, SCL8_MARK, SCL4_MARK,
 	HCTS2_N_MARK, SCIFB2_CTS_N_MARK, ATAWR1_N_MARK,
@@ -690,11 +689,11 @@ enum {
 	VI0_G6_MARK, VI2_CLK_MARK,
 	VI0_G7_MARK, VI2_DATA0_MARK,
 	VI0_R0_MARK, VI2_DATA1_MARK,
-	TS_SDATA0_C_MARK, ATACS11_N_MARK,
+	ATACS11_N_MARK,
 	VI0_R1_MARK, VI2_DATA2_MARK,
-	TS_SCK0_C_MARK, ATAG1_N_MARK,
-	VI0_R2_MARK, VI2_DATA3_MARK, TS_SDEN0_C_MARK,
-	VI0_R3_MARK, VI2_DATA4_MARK, TS_SPSYNC0_C_MARK,
+	ATAG1_N_MARK,
+	VI0_R2_MARK, VI2_DATA3_MARK,
+	VI0_R3_MARK, VI2_DATA4_MARK,
 	VI0_R4_MARK, VI2_DATA5_MARK, TX0_C_MARK, SCL1_D_MARK,
 
 	/* IPSR11 */
@@ -702,12 +701,12 @@ enum {
 	VI0_R6_MARK, VI2_DATA7_MARK, TX1_C_MARK, SCL4_B_MARK,
 	VI0_R7_MARK, RX1_C_MARK, CAN0_RX_E_MARK,
 	SDA4_B_MARK, HRX1_D_MARK, SCIFB0_RXD_D_MARK,
-	VI1_HSYNC_N_MARK, AVB_RXD0_MARK, TS_SDATA0_B_MARK,
+	VI1_HSYNC_N_MARK, AVB_RXD0_MARK,
 	TX4_B_MARK, SCIFA4_TXD_B_MARK,
-	VI1_VSYNC_N_MARK, AVB_RXD1_MARK, TS_SCK0_B_MARK,
+	VI1_VSYNC_N_MARK, AVB_RXD1_MARK,
 	RX4_B_MARK, SCIFA4_RXD_B_MARK,
-	VI1_CLKENB_MARK, AVB_RXD2_MARK, TS_SDEN0_B_MARK,
-	VI1_FIELD_MARK, AVB_RXD3_MARK, TS_SPSYNC0_B_MARK,
+	VI1_CLKENB_MARK, AVB_RXD2_MARK,
+	VI1_FIELD_MARK, AVB_RXD3_MARK,
 	VI1_CLK_MARK, AVB_RXD4_MARK, VI1_DATA0_MARK, AVB_RXD5_MARK,
 	VI1_DATA1_MARK, AVB_RXD6_MARK, VI1_DATA2_MARK, AVB_RXD7_MARK,
 	VI1_DATA3_MARK, AVB_RX_ER_MARK, VI1_DATA4_MARK, AVB_MDIO_MARK,
@@ -1026,24 +1025,20 @@ static const u16 pinmux_data[] = {
 	PINMUX_IPSR_MSEL(IP4_27_26, MSIOF2_SCK_D, SEL_SOF2_3),
 	PINMUX_IPSR_DATA(IP4_30_28, SSI_SCK5),
 	PINMUX_IPSR_MSEL(IP4_30_28, MSIOF1_SCK_C, SEL_SOF1_2),
-	PINMUX_IPSR_MSEL(IP4_30_28, TS_SDATA0, SEL_TSIF0_0),
 	PINMUX_IPSR_MSEL(IP4_30_28, MSIOF2_SYNC_D, SEL_SOF2_3),
 	PINMUX_IPSR_DATA(IP4_30_28, VI1_R2_B),
 
 	/* IPSR5 */
 	PINMUX_IPSR_DATA(IP5_2_0, SSI_WS5),
 	PINMUX_IPSR_MSEL(IP5_2_0, MSIOF1_SYNC_C, SEL_SOF1_2),
-	PINMUX_IPSR_MSEL(IP5_2_0, TS_SCK0, SEL_TSIF0_0),
 	PINMUX_IPSR_MSEL(IP5_2_0, MSIOF2_TXD_D, SEL_SOF2_3),
 	PINMUX_IPSR_DATA(IP5_2_0, VI1_R3_B),
 	PINMUX_IPSR_DATA(IP5_5_3, SSI_SDATA5),
 	PINMUX_IPSR_MSEL(IP5_5_3, MSIOF1_TXD_C, SEL_SOF1_2),
-	PINMUX_IPSR_MSEL(IP5_5_3, TS_SDEN0, SEL_TSIF0_0),
 	PINMUX_IPSR_MSEL(IP5_5_3, MSIOF2_SS1_D, SEL_SOF2_3),
 	PINMUX_IPSR_DATA(IP5_5_3, VI1_R4_B),
 	PINMUX_IPSR_DATA(IP5_8_6, SSI_SCK6),
 	PINMUX_IPSR_MSEL(IP5_8_6, MSIOF1_RXD_C, SEL_SOF1_2),
-	PINMUX_IPSR_MSEL(IP5_8_6, TS_SPSYNC0, SEL_TSIF0_0),
 	PINMUX_IPSR_MSEL(IP5_8_6, MSIOF2_RXD_D, SEL_SOF2_3),
 	PINMUX_IPSR_DATA(IP5_8_6, VI1_R5_B),
 	PINMUX_IPSR_DATA(IP5_11_9, SSI_WS6),
@@ -1228,19 +1223,15 @@ static const u16 pinmux_data[] = {
 	PINMUX_IPSR_DATA(IP9_20_19, VI0_CLKENB),
 	PINMUX_IPSR_MSEL(IP9_20_19, TX4, SEL_SCIF4_0),
 	PINMUX_IPSR_MSEL(IP9_20_19, SCIFA4_TXD, SEL_SCIFA4_0),
-	PINMUX_IPSR_MSEL(IP9_20_19, TS_SDATA0_D, SEL_TSIF0_3),
 	PINMUX_IPSR_DATA(IP9_22_21, VI0_FIELD),
 	PINMUX_IPSR_MSEL(IP9_22_21, RX4, SEL_SCIF4_0),
 	PINMUX_IPSR_MSEL(IP9_22_21, SCIFA4_RXD, SEL_SCIFA4_0),
-	PINMUX_IPSR_MSEL(IP9_22_21, TS_SCK0_D, SEL_TSIF0_3),
 	PINMUX_IPSR_DATA(IP9_24_23, VI0_HSYNC_N),
 	PINMUX_IPSR_MSEL(IP9_24_23, TX5, SEL_SCIF5_0),
 	PINMUX_IPSR_MSEL(IP9_24_23, SCIFA5_TXD, SEL_SCIFA5_0),
-	PINMUX_IPSR_MSEL(IP9_24_23, TS_SDEN0_D, SEL_TSIF0_3),
 	PINMUX_IPSR_DATA(IP9_26_25, VI0_VSYNC_N),
 	PINMUX_IPSR_MSEL(IP9_26_25, RX5, SEL_SCIF5_0),
 	PINMUX_IPSR_MSEL(IP9_26_25, SCIFA5_RXD, SEL_SCIFA5_0),
-	PINMUX_IPSR_MSEL(IP9_26_25, TS_SPSYNC0_D, SEL_TSIF0_3),
 	PINMUX_IPSR_DATA(IP9_28_27, VI0_DATA3_VI0_B3),
 	PINMUX_IPSR_MSEL(IP9_28_27, SCIF3_SCK_B, SEL_SCIF3_1),
 	PINMUX_IPSR_MSEL(IP9_28_27, SCIFA3_SCK_B, SEL_SCIFA3_1),
@@ -1286,18 +1277,14 @@ static const u16 pinmux_data[] = {
 	PINMUX_IPSR_DATA(IP10_18_17, VI2_DATA0),
 	PINMUX_IPSR_DATA(IP10_21_19, VI0_R0),
 	PINMUX_IPSR_DATA(IP10_21_19, VI2_DATA1),
-	PINMUX_IPSR_MSEL(IP10_21_19, TS_SDATA0_C, SEL_TSIF0_2),
 	PINMUX_IPSR_DATA(IP10_21_19, ATACS11_N),
 	PINMUX_IPSR_DATA(IP10_24_22, VI0_R1),
 	PINMUX_IPSR_DATA(IP10_24_22, VI2_DATA2),
-	PINMUX_IPSR_MSEL(IP10_24_22, TS_SCK0_C, SEL_TSIF0_2),
 	PINMUX_IPSR_DATA(IP10_24_22, ATAG1_N),
 	PINMUX_IPSR_DATA(IP10_26_25, VI0_R2),
 	PINMUX_IPSR_DATA(IP10_26_25, VI2_DATA3),
-	PINMUX_IPSR_MSEL(IP10_26_25, TS_SDEN0_C, SEL_TSIF0_2),
 	PINMUX_IPSR_DATA(IP10_28_27, VI0_R3),
 	PINMUX_IPSR_DATA(IP10_28_27, VI2_DATA4),
-	PINMUX_IPSR_MSEL(IP10_28_27, TS_SPSYNC0_C, SEL_TSIF0_2),
 	PINMUX_IPSR_DATA(IP10_31_29, VI0_R4),
 	PINMUX_IPSR_DATA(IP10_31_29, VI2_DATA5),
 	PINMUX_IPSR_MSEL(IP10_31_29, TX0_C, SEL_SCIF0_2),
@@ -1320,20 +1307,16 @@ static const u16 pinmux_data[] = {
 	PINMUX_IPSR_MSEL(IP11_8_6, SCIFB0_RXD_D, SEL_SCIFB_3),
 	PINMUX_IPSR_MSEL(IP11_11_9, VI1_HSYNC_N, SEL_VI1_0),
 	PINMUX_IPSR_DATA(IP11_11_9, AVB_RXD0),
-	PINMUX_IPSR_MSEL(IP11_11_9, TS_SDATA0_B, SEL_TSIF0_1),
 	PINMUX_IPSR_MSEL(IP11_11_9, TX4_B, SEL_SCIF4_1),
 	PINMUX_IPSR_MSEL(IP11_11_9, SCIFA4_TXD_B, SEL_SCIFA4_1),
 	PINMUX_IPSR_MSEL(IP11_14_12, VI1_VSYNC_N, SEL_VI1_0),
 	PINMUX_IPSR_DATA(IP11_14_12, AVB_RXD1),
-	PINMUX_IPSR_MSEL(IP11_14_12, TS_SCK0_B, SEL_TSIF0_1),
 	PINMUX_IPSR_MSEL(IP11_14_12, RX4_B, SEL_SCIF4_1),
 	PINMUX_IPSR_MSEL(IP11_14_12, SCIFA4_RXD_B, SEL_SCIFA4_1),
 	PINMUX_IPSR_MSEL(IP11_16_15, VI1_CLKENB, SEL_VI1_0),
 	PINMUX_IPSR_DATA(IP11_16_15, AVB_RXD2),
-	PINMUX_IPSR_MSEL(IP11_16_15, TS_SDEN0_B, SEL_TSIF0_1),
 	PINMUX_IPSR_MSEL(IP11_18_17, VI1_FIELD, SEL_VI1_0),
 	PINMUX_IPSR_DATA(IP11_18_17, AVB_RXD3),
-	PINMUX_IPSR_MSEL(IP11_18_17, TS_SPSYNC0_B, SEL_TSIF0_1),
 	PINMUX_IPSR_MSEL(IP11_19, VI1_CLK, SEL_VI1_0),
 	PINMUX_IPSR_DATA(IP11_19, AVB_RXD4),
 	PINMUX_IPSR_MSEL(IP11_20, VI1_DATA0, SEL_VI1_0),
@@ -5535,7 +5518,7 @@ static const struct pinmux_cfg_reg pinmux_config_regs[] = {
 		/* IP4_31 [1] */
 		0, 0,
 		/* IP4_30_28 [3] */
-		FN_SSI_SCK5, FN_MSIOF1_SCK_C, FN_TS_SDATA0, 0,
+		FN_SSI_SCK5, FN_MSIOF1_SCK_C, 0, 0,
 		FN_MSIOF2_SYNC_D, FN_VI1_R2_B,
 		0, 0,
 		/* IP4_27_26 [2] */
@@ -5598,15 +5581,15 @@ static const struct pinmux_cfg_reg pinmux_config_regs[] = {
 		FN_SSI_WS6, 0, FN_MSIOF2_SS2_D, FN_VI1_R6_B,
 		0, 0, 0, 0,
 		/* IP5_8_6 [3] */
-		FN_SSI_SCK6, FN_MSIOF1_RXD_C, FN_TS_SPSYNC0, 0,
+		FN_SSI_SCK6, FN_MSIOF1_RXD_C, 0, 0,
 		FN_MSIOF2_RXD_D, FN_VI1_R5_B,
 		0, 0,
 		/* IP5_5_3 [3] */
-		FN_SSI_SDATA5, FN_MSIOF1_TXD_C, FN_TS_SDEN0, 0,
+		FN_SSI_SDATA5, FN_MSIOF1_TXD_C, 0, 0,
 		FN_MSIOF2_SS1_D, FN_VI1_R4_B,
 		0, 0,
 		/* IP5_2_0 [3] */
-		FN_SSI_WS5, FN_MSIOF1_SYNC_C, FN_TS_SCK0, 0,
+		FN_SSI_WS5, FN_MSIOF1_SYNC_C, 0, 0,
 		FN_MSIOF2_TXD_D, FN_VI1_R3_B,
 		0, 0, }
 	},
@@ -5742,13 +5725,13 @@ static const struct pinmux_cfg_reg pinmux_config_regs[] = {
 		/* IP9_28_27 [2] */
 		FN_VI0_DATA3_VI0_B3, FN_SCIF3_SCK_B, FN_SCIFA3_SCK_B, 0,
 		/* IP9_26_25 [2] */
-		FN_VI0_VSYNC_N, FN_RX5, FN_SCIFA5_RXD, FN_TS_SPSYNC0_D,
+		FN_VI0_VSYNC_N, FN_RX5, FN_SCIFA5_RXD, 0,
 		/* IP9_24_23 [2] */
-		FN_VI0_HSYNC_N, FN_TX5, FN_SCIFA5_TXD, FN_TS_SDEN0_D,
+		FN_VI0_HSYNC_N, FN_TX5, FN_SCIFA5_TXD, 0,
 		/* IP9_22_21 [2] */
-		FN_VI0_FIELD, FN_RX4, FN_SCIFA4_RXD, FN_TS_SCK0_D,
+		FN_VI0_FIELD, FN_RX4, FN_SCIFA4_RXD, 0,
 		/* IP9_20_19 [2] */
-		FN_VI0_CLKENB, FN_TX4, FN_SCIFA4_TXD, FN_TS_SDATA0_D,
+		FN_VI0_CLKENB, FN_TX4, FN_SCIFA4_TXD, 0,
 		/* IP9_18_17 [2] */
 		FN_DU1_CDE, 0, FN_PWM4_B, 0,
 		/* IP9_16 [1] */
@@ -5783,15 +5766,15 @@ static const struct pinmux_cfg_reg pinmux_config_regs[] = {
 		FN_VI0_R4, FN_VI2_DATA5, 0, FN_TX0_C, FN_SCL1_D,
 		0, 0, 0,
 		/* IP10_28_27 [2] */
-		FN_VI0_R3, FN_VI2_DATA4, 0, FN_TS_SPSYNC0_C,
+		FN_VI0_R3, FN_VI2_DATA4, 0, 0,
 		/* IP10_26_25 [2] */
-		FN_VI0_R2, FN_VI2_DATA3, 0, FN_TS_SDEN0_C,
+		FN_VI0_R2, FN_VI2_DATA3, 0, 0,
 		/* IP10_24_22 [3] */
-		FN_VI0_R1, FN_VI2_DATA2, 0, FN_TS_SCK0_C, FN_ATAG1_N,
+		FN_VI0_R1, FN_VI2_DATA2, 0, 0, FN_ATAG1_N,
 		0, 0, 0,
 		/* IP10_21_19 [3] */
 		FN_VI0_R0, FN_VI2_DATA1, 0,
-		FN_TS_SDATA0_C, FN_ATACS11_N,
+		0, FN_ATACS11_N,
 		0, 0, 0,
 		/* IP10_18_17 [2] */
 		FN_VI0_G7, FN_VI2_DATA0, 0, 0,
@@ -5840,15 +5823,15 @@ static const struct pinmux_cfg_reg pinmux_config_regs[] = {
 		/* IP11_19 [1] */
 		FN_VI1_CLK, FN_AVB_RXD4,
 		/* IP11_18_17 [2] */
-		FN_VI1_FIELD, FN_AVB_RXD3, FN_TS_SPSYNC0_B, 0,
+		FN_VI1_FIELD, FN_AVB_RXD3, 0, 0,
 		/* IP11_16_15 [2] */
-		FN_VI1_CLKENB, FN_AVB_RXD2, FN_TS_SDEN0_B, 0,
+		FN_VI1_CLKENB, FN_AVB_RXD2, 0, 0,
 		/* IP11_14_12 [3] */
-		FN_VI1_VSYNC_N, FN_AVB_RXD1, FN_TS_SCK0_B,
+		FN_VI1_VSYNC_N, FN_AVB_RXD1, 0,
 		FN_RX4_B, FN_SCIFA4_RXD_B,
 		0, 0, 0,
 		/* IP11_11_9 [3] */
-		FN_VI1_HSYNC_N, FN_AVB_RXD0, FN_TS_SDATA0_B,
+		FN_VI1_HSYNC_N, FN_AVB_RXD0, 0,
 		FN_TX4_B, FN_SCIFA4_TXD_B,
 		0, 0, 0,
 		/* IP11_8_6 [3] */
@@ -6109,8 +6092,8 @@ static const struct pinmux_cfg_reg pinmux_config_regs[] = {
 		FN_SEL_TMU1_0, FN_SEL_TMU1_1,
 		/* SEL_LBS [2] */
 		FN_SEL_LBS_0, FN_SEL_LBS_1, FN_SEL_LBS_2, FN_SEL_LBS_3,
-		/* SEL_TSIF0 [2] */
-		FN_SEL_TSIF0_0, FN_SEL_TSIF0_1, FN_SEL_TSIF0_2, FN_SEL_TSIF0_3,
+		/* RESERVED [2] */
+		0, 0, 0, 0,
 		/* SEL_SOF0 [2] */
 		FN_SEL_SOF0_0, FN_SEL_SOF0_1, FN_SEL_SOF0_2, 0, }
 	},
-- 
2.7.4

